<launch>
  <node name="celex_zynq_callback" pkg="celex5_zynq" type="celex5_zynq_callback_node" output="screen" >
    <param name="celex_mode" value="Event_Address_Only_Mode" />
    <param name="threshold" value="170" />
    <param name="clock_rate" value="100" />
  </node>
</launch>